vector/src/main/java/im/vector/app/features/home/room/detail/composer/voice/VoiceMessageViews.kt

```diff
@@ -69,6 +83,8 @@ class VoiceMessageViews(
         observeMicButton(actions)
     }
 
+    private fun getTouchedPositionPercentage(motionEvent: MotionEvent, view: View) = motionEvent.x / view.width
```

Comment:
Not tested, but show probably use `coerceIn(0,1)` to avoid overflow.

Reply:
More secure, done.